test/components/views/elements/AccessibleButton-test.tsx

```diff
@@ -38,168 +36,136 @@ describe("<AccessibleButton />", () => {
         unmockPlatformPeg();
     });
 
-    const makeKeyboardEvent = (key: string) =>
-        ({
-            key,
-            stopPropagation: jest.fn(),
-            preventDefault: jest.fn(),
-        } as unknown as KeyboardEvent);
-
     it("renders div with role button by default", () => {
-        const component = getComponent();
-        expect(component).toMatchSnapshot();
+        const { asFragment } = getComponent();
+        expect(asFragment()).toMatchSnapshot();
     });
 
     it("renders a button element", () => {
-        const component = getComponent({ element: "button" });
-        expect(component).toMatchSnapshot();
+        const { asFragment } = getComponent({ element: "button" });
+        expect(asFragment()).toMatchSnapshot();
     });
 
     it("renders with correct classes when button has kind", () => {
-        const component = getComponent({
+        const { asFragment } = getComponent({
             kind: "primary",
         });
-        expect(component).toMatchSnapshot();
+        expect(asFragment()).toMatchSnapshot();
     });
 
     it("disables button correctly", () => {
         const onClick = jest.fn();
-        const component = getComponent({
+        const { container } = getComponent({
             onClick,
             disabled: true,
         });
-        expect(component.find(".mx_AccessibleButton").props().disabled).toBeTruthy();
-        expect(component.find(".mx_AccessibleButton").props()["aria-disabled"]).toBeTruthy();
+        expect(container.querySelector("[disabled]")).toBeTruthy();
+        expect(container.querySelector("[aria-disabled]")).toBeTruthy();
 
-        act(() => {
-            component.simulate("click");
-        });
+        const btn = container.querySelector(".mx_AccessibleButton")!;
```

Comment:
I think the RTL way is to search the button by label.
Can we change it and add a label to the button and select it here (+other places).